.github/workflows/deploy.yml

```
@@ -0,0 +1,61 @@
+name: Deploy Rails API to ECS
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)